### PR TITLE
Add invite comments

### PR DESCRIPTION
**Pull request by ClearlyClaire**

### CODE REVIEW

**Comment by Gargron** [comment]
Was someone asking for this?

**Comment by ThibG** [comment]
Yeah

File: app/models/invite.rb [hunk 1]
```diff
@@ -12,6 +12,7 @@
 #  created_at :datetime         not null
 #  updated_at :datetime         not null
 #  autofollow :boolean          default(FALSE), not null
+#  note       :text
```
Comment (Gargron) [inline app/models/invite.rb]: So is it a `comment` or a `note`?

**Review by Gargron — CHANGES_REQUESTED** [review]

**Comment by ThibG** [comment]
I think the reason why this hadn't been merged was to not add a new translatable string just before an upcoming release? If so, this could be merged now

**Comment by ThibG** [comment]
@Gargron can we get that in before the 3.0 call for translations?

**Review by Gargron — APPROVED** [review]

**Pull request merged**